Mage.Sets/src/mage/cards/r/RipplesOfPotential.java

```diff
@@ -0,0 +1,133 @@
+package mage.cards.r;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.counter.ProliferateEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.WatcherScope;
+import mage.filter.FilterPermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.permanent.PermanentIdPredicate;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.TargetPermanent;
+import mage.watchers.Watcher;
+
+/**
+ * Ripples of Potential {1}{U}
+ * Instant
+ * Proliferate, then choose any number of permanents you control that had a counter put on them this way. Those permanents phase out.
+ *
+ * @author DominionSpy
+ */
+public class RipplesOfPotential extends CardImpl {
+
+    public RipplesOfPotential(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{1}{U}");
+
+        // Proliferate, then choose any number of permanents you control that had a counter put on them this way. Those permanents phase out.
+        // A watcher is required as another effect may prevent counters from being put on permanents (e.g. Solemnity)
+        this.getSpellAbility().addEffect(new ProliferateEffect());
+        this.getSpellAbility().addEffect(new RipplesOfPotentialEffect());
+        this.getSpellAbility().addWatcher(new RipplesOfPotentialWatcher());
+    }
+
+    private RipplesOfPotential(final RipplesOfPotential card) {
+        super(card);
+    }
+
+    @Override
+    public RipplesOfPotential copy() {
+        return new RipplesOfPotential(this);
+    }
+}
+
+class RipplesOfPotentialEffect extends OneShotEffect {
+
+    public RipplesOfPotentialEffect() {
+        super(Outcome.Benefit);
+        staticText = "Proliferate, then choose any number of permanents you control that had a counter put on them this way. Those permanents phase out.";
```

Comment:
"Proliferate" is handled by the ProliferateEffect, so start this text with ", then choose"